src/services/authenticationLockService.ts

```diff
@@ -1,139 +0,0 @@
-/**
- * This service retrieves and updates the user profile from the API system using
- * an API client.
- */
-
-import { TableClient, TransactionAction, odata } from "@azure/data-tables";
-import { DateFromString } from "@pagopa/ts-commons/lib/dates";
-import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
-import * as ROA from "fp-ts/ReadonlyArray";
-import * as TE from "fp-ts/TaskEither";
-import * as E from "fp-ts/lib/Either";
-import { flow, identity, pipe } from "fp-ts/lib/function";
-import * as t from "io-ts";
-
-import { UnlockCode } from "../../generated/session/UnlockCode";
-import * as AI from "../utils/AsyncIterableTask";
-import { errorsToError } from "../utils/errorsFormatter";
-
-export type NotReleasedAuthenticationLockData = t.TypeOf<
-  typeof NotReleasedAuthenticationLockData
->;
-const NotReleasedAuthenticationLockData = t.type({
-  partitionKey: FiscalCode,
-  rowKey: UnlockCode,
-
-  // eslint-disable-next-line sort-keys
-  CreatedAt: DateFromString
-});
-
-export default class AuthenticationLockService {
```

Comment:
Since the service is no longer used, we can remove the mocked version too.

Reply:
thank you, done in https://github.com/pagopa/io-backend/pull/1224/commits/97d24e614f9f18fef6e0a874829ba795aabbf64e